spec/factories/submission.rb

```diff
@@ -2,6 +2,7 @@
   factory :submission do
     framework
     supplier
+    association :created_by, factory: :user
```

Comment:
As a general rule I think unless an association is actually required or essential to the model, it shouldn't be included in the base factory to keep unnecessary record creation to a minimum.